.github/workflows/docs.yml

```diff
@@ -0,0 +1,40 @@
+name: Auto Doc Gen
+on:
+  push:
+    branches:
+      - master
+
+env:
+  NODE_VERSION: 14.x
```

Comment:
not super important but is Node 16 available for this?

Reply:
I'll use 16.x. Was using the latest one defined on the other workflow, but Bryn just updated it to 16.x.

Reply:
Thanks!